Resources/translations/messages.fr.xliff

```diff
@@ -14,112 +14,6 @@
-                <target>E-Mail</target>
-            </trans-unit>
-            <trans-unit id="16">
-                <source>l91_sulu_form.type.phone</source>
-                <target>Téléphone</target>
-            </trans-unit>
-            <trans-unit id="17">
-                <source>l91_sulu_form.type.fax</source>
-                <target>Fax</target>
-            </trans-unit>
-            <trans-unit id="18">
-                <source>l91_sulu_form.type.street</source>
-                <target>Rue</target>
-            </trans-unit>
-            <trans-unit id="19">
-                <source>l91_sulu_form.type.zip</source>
-                <target>Code postal</target>
-            </trans-unit>
-            <trans-unit id="20">
-                <source>l91_sulu_form.type.city</source>
-                <target>Ville</target>
-            </trans-unit>
-            <trans-unit id="21">
-                <source>l91_sulu_form.type.state</source>
-                <target>Etat</target>
-            </trans-unit>
-            <trans-unit id="22">
-                <source>l91_sulu_form.type.country</source>
-                <target>Pays</target>
-            </trans-unit>
-            <trans-unit id="23">
-                <source>l91_sulu_form.type.function</source>
-                <target>Fonction</target>
-            </trans-unit>
-            <trans-unit id="24">
-                <source>l91_sulu_form.type.company</source>
-                <target>Compagnie</target>
-            </trans-unit>
-            <trans-unit id="25">
-                <source>l91_sulu_form.type.text</source>
-                <target>Simple Textfield</target>
-            </trans-unit>
-            <trans-unit id="26">
-                <source>l91_sulu_form.type.textarea</source>
-                <target>Multiline Textfield</target>
-            </trans-unit>
-            <trans-unit id="27">
-                <source>l91_sulu_form.type.date</source>
-                <target>Date</target>
-            </trans-unit>
-            <trans-unit id="28">
-                <source>l91_sulu_form.type.headline</source>
-                <target>Gros titre</target>
-            </trans-unit>
-            <trans-unit id="29">
-                <source>l91_sulu_form.type.attachment</source>
-                <target>Attachement</target>
-            </trans-unit>
-            <trans-unit id="30">
-                <source>l91_sulu_form.type.checkbox</source>
-                <target>Checkbox</target>
-            </trans-unit>
-            <trans-unit id="31">
-                <source>l91_sulu_form.type.checkboxMultiple</source>
-                <target>Checkboxes</target>
-            </trans-unit>
-            <trans-unit id="32">
-                <source>l91_sulu_form.type.dropdown</source>
-                <target>Select</target>
-            </trans-unit>
-            <trans-unit id="33">
-                <source>l91_sulu_form.type.dropdownMultiple</source>
```

Comment:
as i see it was added for the FormControlller `trans` function did include there to use the `backend` domain so this translates dont need to be duplicated